mailscanner/functions.php

```diff
@@ -4682,9 +4689,8 @@ function printTenMinutesGraph()
      SELECT
       timestamp AS xaxis,
       1 as total_mail,
-      virusinfected AS total_virus,
-      isspam AS total_spam,
-      size AS total_size
+      sum(virusinfected+nameinfected+otherinfected) AS total_virus,
+      sum(isspam+ishighspam+issaspam+isrblspam+spamblacklisted) AS total_spam
```

Comment:
you have to revert total_spam as before your edit, and you can use a `CASE` in total_virus:
```sql
SUM(CASE WHEN (virusinfected+nameinfected+otherinfected) > 0 THEN 1 ELSE 0 END) AS total_virus
```